migreringer/src/main/resources/db/migration/defaultDS/3.1/V3.1_01__TFP-5571-lokalt_oppgave_tabelle.sql

```diff
@@ -37,5 +41,11 @@ comment on column OPPGAVE.ENHET is 'Tildelt enhet som skal løse oppgaven.'
 /
 comment on column OPPGAVE.FRIST is 'Frist dato for å løse oppgaven.'
```

Comment:
Finnes det i dag for disse type oppgaver? Hvordan skal det utledes?

Reply:
Den settes til dagens dato faktisk - tror vi bare gjør det samme i første omgang